src/lib/Behat/Page/UserSettingsPage.php

```diff
@@ -50,9 +54,30 @@ protected function specifyLocators(): array
     {
         return [
             new VisibleCSSLocator('button', '.ibexa-btn'),
+            new VisibleCSSLocator('title', '.ibexa-edit-header__title'),
+            new VisibleCSSLocator('autosaveDraftEditButton', '#ibexa-tab-my-preferences > div:nth-child(2) > div div.ibexa-table-header__actions > a'),
```

Comment:
I'm not a fan of this selector, looks like it could easily break when new Settings are added.

Maybe we can do it another way:
```

$this->getHTMLPage()
    ->findAll(new VisibleCSSLocator('settingsSection', '#ibexa-tab-my-preferences .ibexa-details')
    ->getBy(new ChildElementTextCriterion(new VisibleCSSLocator('settingHeader', '.ibexa-table-header'), 'Edit Content')
    ->find(new VisibleCSSLocator('editButton', '[data-bs-original-title="Edit"]'))
    ->click()
```